railties/test/application/rake/multi_dbs_test.rb

```diff
@@ -594,11 +594,11 @@ def change
         db_migrate_and_schema_dump_and_load_one_database("animals", "ruby")
       end
 
-      test "db:migrate:name and db:structure:dump:name and db:structure:load:name works for the primary database" do
+      test "db:migrate:name and db:schema:dump:name and db:schema:load:name works for the primary database" do
```

Comment:
These test names conflict with the above two tests.

https://buildkite.com/rails/rails/builds/83580#61110966-bc7f-4c64-b165-c61b5e5c882a/1202-1240

Reply:
👍